pmemkv/src/main/java/site/ycsb/db/pmemkv/PmemKVClient.java

```diff
@@ -119,31 +121,60 @@ public void init() throws DBException {
         String engineName = props.getProperty(ENGINE_PROPERTY, "cmap");
 
         String path = props.getProperty(PATH_PROPERTY);
-        if (path == null) {
-          throw new DBException(PATH_PROPERTY + " is obligatory to run PmemKV client");
-        }
         String size = props.getProperty(SIZE_PROPERTY);
-        if (size == null) {
-          throw new DBException(SIZE_PROPERTY + " is obligatory to run PmemKV client");
+        String jsonConfigProp = props.getProperty(JSON_CONFIG_PROPERTY);
+        Path jsonConfigFile = null;
+        String jsonConfigContent = null;
```

Comment:
Why you did that?
It will make garbage.
```
Path jsonConfigFile
String jsonConfigContent
```
is enough.

Reply:
maybe it should be enough, but I keep getting (I guess due to compiling with some `-Xlint` option):
```
.../PmemKVClient.java:[146,15] variable jsonConfigContent might not have been initialized
```
and I've added all these just to be safe. Nonetheless, I've removed redundant ones.

BTW, `db = null` was here in the code from the very beginning, so I guess we all love C/C++ deep in our hearts 😄

Reply:
xD

Reply:
If lint says that, so is true then and we should follow his guidelines :D

Reply:
it only says it just for this one case for some reason, though...

anyway, as I said - I left only the one that's needed.